### PR TITLE
[SPARK-11052] Spaces in the build dir causes failures in the build/mv…

### PR DESCRIPTION
**Pull request by trystanleftwich**
…n script

### CODE REVIEW

**Comment by JoshRosen** [comment]
This looks reasonable to me. It reminds me: it would be nice to run http://www.shellcheck.net/ on these scripts, since that tool can find more of these types of quoting errors. Even cooler would be integrating it into our build as a linter!

File: make-distribution.sh [hunk 1]
```diff
@@ -121,7 +121,7 @@ if [ $(command -v git) ]; then
 fi
 
 
-if [ ! $(command -v "$MVN") ] ; then
+if [ ! "$(command -v "$MVN")" ] ; then
```
Comment (srowen) [inline make-distribution.sh]: Just making sure this works as expected with quotes within quotes? Otherwise LGTM

Comment (trystanleftwich) [inline make-distribution.sh]: It does, here is the output from that line when it has quotes:
+ '[' '!' '/tmp/test spaces/spark/build/mvn' ']'


Before:
+ '[' '!' /tmp/test spaces/spark/build/mvn ']'

Comment (srowen) [inline make-distribution.sh]: Yeah I tried it locally too. Just making sure the first two double qutoes didn't somehow bind more closely than the parens.

**Pull request closed without merging**